server/utils/rateLimit.js

```diff
@@ -0,0 +1,8 @@
+const Bottleneck = require('bottleneck');
+const rateLimiter = new Bottleneck({
+  minTime: 1050 / 5,
```

Comment:
Not quite sure what this `minTime` means just by looking at these numbers. If they represent some discrete time units, can we give them names?

Reply:
I removed rate limiting (in response to our discussion yesterday) -- rate limiting is currently not needed at this point of the project since we'll have the client receive the rate limiting 429 message